contracts/modules/grouping/GroupingModule.sol

```diff
@@ -183,13 +192,30 @@ contract GroupingModule is
     /// @param ipIds The IP IDs.
     function claimReward(address groupId, address token, address[] calldata ipIds) external whenNotPaused {
         IGroupRewardPool pool = IGroupRewardPool(GROUP_IP_ASSET_REGISTRY.getGroupRewardPool(groupId));
-        // claim reward from group IPA's RoyaltyVault to group pool
-        pool.collectRoyalties(groupId, token);
         // trigger group pool to distribute rewards to group members vault
         uint256[] memory rewards = pool.distributeRewards(groupId, token, ipIds);
         emit ClaimedReward(groupId, token, ipIds, rewards);
     }
 
+    /// @notice Collects royalties into the pool, making them claimable by group member IPs.
+    /// @param groupId The address of the group.
+    /// @param token The address of the token.
+    /// @param snapshots The snapshot IDs of the royalty vault for the given group to collect royalties.
+    function collectRoyalties(
+        address groupId,
+        address token,
+        uint256[] calldata snapshots
```

Comment:
would rename the input to `snapshotIds` for consistency with ip royalty vault contract

Reply:
renamed.